R/pr.R

```diff
@@ -5,15 +5,15 @@
 #'    anticipation of making a PR.
 #' * `pr_fetch(number)` downloads a remote PR so you can edit locally.
 #' * `pr_push()` pushes local changes to GitHub, after checking that there
-#'    aren't any remote changes you need to retrieve first. It will create
-#'    a PR if needed
-#' * `pr_pull()` retrieves changes from the GitHub PR. Run this if others
-#'    have made suggestions or pushed into your PR.
+#'    aren't any remote changes you need to retrieve first. It will help you
+#'    initiate a PR, if needed.
+#' * `pr_pull()` retrieves changes from the GitHub PR. Run this if the remote PR
+#'    has commits you don't have.
```

Comment:
Reworded because it's not always "your PR", i.e. account for the maintainer's use case.